travis/prepare/prepare-mongodb

```diff
@@ -1,11 +1,47 @@
 #!/bin/bash
-echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
-if [ -f $SYLIUS_CACHE_DIR/composer-mongodb.lock ]; then
-    cp $SYLIUS_CACHE_DIR/composer-mongodb.lock composer.lock
-else
-    composer require doctrine/mongodb-odm="1.0.*@dev" --no-update
-    composer require jmikola/geojson="~1.0" --no-update
-    composer update doctrine/mongodb-odm jmikola/geojson --prefer-source --no-interaction
+function main
+{
+    echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+
+    if is_cache_fresh; then
+        restore_composer_lock_from_cache
+    else
+        install_mongodb
+        store_composer_lock_in_cache
+    fi
+}
+
+function install_mongodb
+{
+    composer require doctrine/mongodb-odm="^1.0.2" --no-update
+    composer update doctrine/mongodb-odm --prefer-source --no-interaction
```

Comment:
As you don't call additional require anymore, you can remove this & `--no-update` line above.